### DISCUSSION TITLE
build fat jar of rest client app

### DISCUSSION THREAD

**Discussion by guykatz** [microstream-one/microstream#654] [Q&A] [answered]
Hi guys;
i want to buidl the microstream rest storage client app from scratch into a fat client (like the jar in mvnrepo which is about 55MB in size)
cloning the microstream repo and building everything from scratch from the POM I get this jar without its dependencies. 
is there another pom somewhere that will build a fat jar? whats my best option
i dont use maven in our build system so I am strugling a bit
thanks

**Comment by hg-ms** [comment, marked as answer]
Hi,
the maven profile 'production' should do the job:
`mvn -P production clean package`
